debian/control

```diff
@@ -332,5 +332,6 @@ Description: Backup Archiving Recovery Open Sourced - common tools
 
 
 
+
```

Comment:
I guess, debian/control should not be part of the PR.

Reply:
In fact, I think this change is required.